apps/qubit/modules/informationobject/templates/_cardViewResults.php

```diff
@@ -23,23 +20,27 @@
             ['alt' => isset($doc['digitalObject']['digitalObjectAltText']) ? $doc['digitalObject']['digitalObjectAltText'] : truncate_text(strip_markdown($title), 100)]),
             ['module' => 'informationobject', 'slug' => $doc['slug']]); ?>
 
-        <?php } elseif (isset($doc['digitalObject']) && !empty($doc['digitalObject']['mediaTypeId'])) { ?>
+        <?php } else { ?>
 
           <?php echo link_to(image_tag(QubitDigitalObject::getGenericIconPathByMediaTypeId($doc['digitalObject']['mediaTypeId']),
```

Comment:
Without the check over `$doc['digitalObject']['mediaTypeId']` this could raise a `PHP Notice:  Undefined index: mediaTypeId`.